### PR TITLE
Updates from gaf

### PR DESCRIPTION
**Pull request by Chris7**
Allows the use of a csv file and alignments via gaf files to add new nodes to a graph.

### CODE REVIEW

File: src/exports/gfa.rs [hunk 1]
```diff
@@ -57,7 +57,8 @@ pub fn export_gfa(
 
     let mut edges = edge_set.into_iter().collect();
 
-    let blocks = Edge::blocks_from_edges(conn, &edges);
+    let mut blocks = Edge::blocks_from_edges(conn, &edges);
+    blocks.sort_by(|a, b| a.node_id.cmp(&b.node_id));
```
Comment (Chris7) [inline src/exports/gfa.rs:61]: this is cosmetic to make the graph output ordered consistently.

File: docs/updates_with_gaf.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+>simple_insert_right
+acacagaaaaaggcaggcagagaaaataacaaggataaagacactgaagt
+```
+
+This file can then be fed to an aligner. Here are several examples:
+
+First, we need to export to a common graph format, GFA
+
+```console
+gen --db example.db export -n k562 -g k562.gfa
```
Comment (dkhofer) [inline docs/updates_with_gaf.md:51]: Could you say something about what file was imported to create this database?

File: src/updates/gaf.rs [hunk 3]
```diff
@@ -0,0 +1,555 @@
...
+        let end_node_id = graph
+            .nodes()
+            .filter(|node| node.node_id == 1001)
+            .collect::<Vec<GraphNode>>();
+
+        let edges: Vec<(GraphNode, GraphNode, &GraphEdge)> = graph
+            .edges_directed(end_node_id[0], Direction::Outgoing)
+            .collect();
+
+        // This checks that we have an incoming edge from our new insert to the old end of the graph
```
Comment (dkhofer) [inline src/updates/gaf.rs:578]: Should this comment say we have an outgoing edge from the old end of the graph to the new insert?

File: src/updates/gaf.rs [hunk 4]
```diff
@@ -0,0 +1,581 @@
...
+    gaf_path: P,
+    csv_path: P,
+    collection_name: &'a str,
+    sample_name: impl Into<Option<&'a str>>,
+) where
+    P: AsRef<Path> + Clone,
+{
+    // Given a gaf, this will incorporate the alignment into the specified graph, creating new nodes.
+
+    let mut session = session::Session::new(conn).unwrap();
```
Comment (dkhofer) [inline src/updates/gaf.rs:89]: I recently added convenience methods to wrap up the standard start/end operation code, could you use them here? https://github.com/ginkgobioworks/gen/blob/main/src/operation_management.rs#L805

**Comment by dkhofer** [comment]
As discussed offline, please add a child sample flag and add the necessary code for it.  Everything else is minor, LGTM overall, approving

**Review by dkhofer — APPROVED** [review]

**Pull request merged**